samples/subsys/ipc/rpmsg_service/sysbuild.cmake

```diff
@@ -0,0 +1,18 @@
+cmake_minimum_required(VERSION 3.20.0)
+# Copyright (c) 2019 Linaro Limited
+# Copyright (c) 2018-2021 Nordic Semiconductor ASA
+# Copyright (c) 2023 Arm Limited
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+ExternalZephyrProject_Add(
+    APPLICATION rpmsg_service_remote
+    SOURCE_DIR ${APP_DIR}/remote
+    BOARD ${SB_CONFIG_RPMSG_REMOTE_BOARD}
+  )
+
+add_dependencies(rpmsg_service rpmsg_service_remote)
+if(("${SB_BOARD}" STREQUAL "lpcxpresso54114_m4"))
+  sysbuild_add_dependencies(CONFIGURE core_m0_inc_target rpmsg_service_remote)
+endif()
```

Comment:
Can this be removed?
* `SB_BOARD` is undefined, so this code does nothing.
* I don't see `core_m0_inc_target` defined anywhere, neither as a CMake target nor a sysbuild image.